### ISSUE TITLE
Accessibility: Add link form contenteditable element missing label

### ISSUE THREAD

**Issue by samsch** [grafana/grafana#109755]
Add link form url editor does not have an accessible label.

The add link input is a contenteditable element, and simply adding an ID to it and for attribute to the label does not work to connect the label to it. A possible solution is adding aria-labelledby to the editor (which requires hacking that attribute in with useEffect, since Slate doesn’t appear to allow passing aria- attributes or IDs), and putting an ID on the label. Additionally, the label should have a click event listener which focuses the editor.

This **MUST** be tested with a screen reader.

<img width="956" height="687" alt="Image" src="https://github.com/user-attachments/assets/f40479da-05a7-43dd-bd42-44d1c2ad9e0b" />

**Comment by DivyamUp14** [comment]
Hi @samsch, I've opened a PR to fix this issue
I used useEffect to apply aria-labelledby to the contenteditable element as per your suggestion above.
